packages/reputation-miner/ReputationMinerClient.js

```diff
@@ -136,7 +136,11 @@ class ReputationMinerClient {
         }
 
         try {
-          const [branchMask, siblings, value] = await this._miner.getHistoricalProofAndValue(req.params.rootHash, key);
+          const historicalProof = await this._miner.getHistoricalProofAndValue(req.params.rootHash, key);
+          if (historicalProof instanceof Error) {
+            return res.status(400).send({ message: historicalProof.message.replace("Error: ") });
```

Comment:
Is this the intended error message?

Reply:
I'm not sure what you're asking?

Reply:
Is the user going to see `Error: ` as the message?

Reply:
No, it's stripping the 'Error' in the message that is returned from `getHistoricalProofAndValue` (my reasoning being that getting a 400 status code already conveys there was an error, and the other error messages returned by this endpoint don't prefix the message with 'Error:'.